Kudu.Contracts/SiteExtensions/SiteExtensionInfo.cs

```diff
@@ -40,21 +41,38 @@ public SiteExtensionInfo(SiteExtensionInfo info)
             InstalledDateTime = info.InstalledDateTime;
         }
 
-        public SiteExtensionInfo(IPackage package)
+        public SiteExtensionInfo(UIPackageMetadata data)
         {
-            Id = package.Id;
-            Title = package.Title;
+            Id = data.Identity.Id;
+            Title = data.Identity.Id;
```

Comment:
there is no title anymore. property has been removed. and i was told id is same as title.